### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by rbillings**
Created README.md with steps for testing each of the test tracking pages.

### CODE REVIEW

**Comment by rbillings** [comment]
r?

File: README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Shavar list testing using test pages
+Instructions on how to create test profiles: https://wiki.mozilla.org/Services/TrackingProtection/Shavar_Server_-_Testing#SET_CUSTOM_PREFS
```
Comment (rpappalax) [inline README.md:2]: Relying on the (now deprecated) wiki is probably best for readability, but I wonder if relying on this as a source of truth as well as prefs.ini as a source of truth will be problematic maintaining both in the long run

File: README.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### https://mozilla.github.io/tracking-test/ads.html
+1. Create a BASE profile
```
Comment (rpappalax) [inline README.md:5]: The explanation you've given for how to test profiles is very clear.  Do you think we should also add an explanation of what these profiles are used for (for context)?  I recall that was particularly confusing before (i.e. why do we have these pref sets definined in the prefs.ini).  For example, while it's clear from the test name what each page testing, it seems like we should also provide some brief summary of why we use BASE for certain tests, STRICT for others and none for others, so our test methodology is clear.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+2. Restart browser
+3. Navigate to https://mozilla.github.io/tracking-test/ads.html
+
+expected result: fox displays, confirming page is serving correct content
```
Comment (rpappalax) [inline README.md:9]: should we clarify specifically what we mean by "correct content?" (for this and all the places mentioned below)

File: README.md (hunk 1 above)
Comment (rbillings) [reply README.md:2]: That may be a valid concern for the long run, however I don't see it changing anytime soon. Test rail works for details on setting up lists, but the wiki still has the larger picture of the project.

File: README.md (hunk 2 above)
Comment (rbillings) [reply README.md:5]: I think an explanation of what the lists are verifying should be defined elsewhere, in developer documentation. It should be clear that the test profiles are for testing specific lists, and that is shown in their naming convention. Our test methodology should cover all of the list variations and set up steps more than explaining why they are that way [imo].

File: README.md (hunk 3 above)
Comment (rbillings) [reply README.md:9]: The correct content is that the fox displays. The test pages serve limited functionality- they show if the content is correct or not. I don't think this would be the place to explain why or how the test pages are created, only that they are working based on the expected results.

**Comment by rbillings** [comment]
@groovecoder this is ready to merge when you have time - I'd like to get the doc in the repo, and we can edit it in the future if changes are warranted.

**Comment by groovecoder** [comment]
🎉 !
